lib/datasource/orb/index.spec.ts

```diff
@@ -36,7 +36,7 @@ describe(getName(), () => {
       httpMock.scope(baseUrl).post('/graphql-unstable').reply(200, {});
       expect(
         await getPkgReleases({
-          datasource,
+          datasource: OrbDatasource.id,
```

Comment:
So here too, add a module scoped `const datasource = OrbDatasource.id` so we have less changes. 🙃